docs/devsite-help/rest-libraries.md

```diff
@@ -0,0 +1,168 @@
+# HTTP/1.1 + REST libraries
+
+As described in the [library types page](library-types.md), the
+HTTP/1.1 + REST libraries are API-specific libraries with source code in
+the [google-api-dotnet-client repository](https://github.com/googleapis/google-api-dotnet-client),
+generated from [Discovery documents](https://developers.google.com/discovery/v1/getting_started)
+instead of API protobuf representations. API calls are made over HTTP/1.1 with
+JSON content.
+
+These libraries depend heavily on the support libraries of Google.Apis and
+Google.Apis.Core, which have their source code in the same repository.
+
+These libraries are still supported and automatically updated as the API
+surfaces change, but the underlying code and generator is not being actively worked
+on, and only high-impact feature requests are considered.
+
+Some features are not as discoverable as we'd like them to be; this page
+describes the use of these features.
+
+## Service object reuse
+
+Service client classes (e.g. `StorageService`) are thread-safe and reusable.
+Typically your application should create a single instance and use it for all operations.
+
+By default, every instantiation of a service client class creates a new instance of
+[HttpClient](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient),
+and creating many of these is, in general, a bad idea. See, for example,
+[this post](https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/) for why.
+
+Every instantiation of a client class also uses more CPU time and requires more memory allocation,
+which are generally best avoided if possible.
```

Comment:
I'm not sure we'd need to justify why? We don't justify this for GAPICs, so, maybe just: "we recommend you use a single instace of a service client"?

Reply:
This was copied from the FAQ, I believe :) But I agree - deleted. (I think it's fine to include the details of it creating a new HttpClient, but not the paragraph starting at line 30.) We've already got the recommendation on line 23.